### PR TITLE
solved Need some left side space in iPad viewport size issue

### LINKED ISSUE #490: Need some left side space in iPad viewport size
**Describe the bug**
left side should be a little space for entry title in iPad viewport size

**To Reproduce**
1. Go to ['https://wordpress.org/patterns/favorites/'](https://wordpress.org/patterns/favorites/)
2. Go iPad device  
3. See error

**Screenshots**
image

**Smartphone (please complete the following information):**
 - Device: [iPad Air, iPad Mini]

**Additional context**

### PR DESCRIPTION
**Pull request by KAJALwp**
Add spacing in left side for entry title

Fixes #490 

Props[ kajalgohel](https://profiles.wordpress.org/kajalgohel/)

### Screenshots

After applying css 

CleanShot 2022-06-07 at 14 41 13@2x

### CODE REVIEW

File: public_html/wp-content/themes/pattern-directory/css/components/_page.scss [hunk 1]
```diff
@@ -8,7 +8,7 @@ body.page {
 			margin: 2rem auto 1rem;
 			max-width: 960px;
 
-			@include breakpoint( $ms-breakpoint ) {
+			@include breakpoint($ms-breakpoint ) {
```
Comment (adamwoodnz) [inline public_html/wp-content/themes/pattern-directory/css/components/_page.scss:11]: Please revert this style change

File: public_html/wp-content/themes/pattern-directory/css/components/_page.scss [hunk 2]
```diff
@@ -27,7 +35,7 @@ body.page-about {
 	.entry-content {
 		padding-top: ms(4);
 
-		@media ( max-width: $breakpoint-small ) {
+		@media (max-width: $breakpoint-small ) {
```
Comment (adamwoodnz) [inline public_html/wp-content/themes/pattern-directory/css/components/_page.scss:38]: Please revert this style change

File: public_html/wp-content/themes/pattern-directory/css/components/_page.scss [hunk 3]
```diff
@@ -36,5 +44,7 @@ body.page-about {
 				padding: 48px !important; // override the padding from the editor.
 			}
 		}
+
 	}
-}
+
+}
```
Comment (adamwoodnz) [inline public_html/wp-content/themes/pattern-directory/css/components/_page.scss:50]: Please revert these changes

**Comment by KAJALwp** [comment]
Hello @adamwoodnz,

I have created new pull request for this issue. Please check it and let me know if there is anything.
PR: #502 

Thanks
